flake-modules/tests.nix

```diff
@@ -39,6 +40,17 @@
           nixvim = self;
         })
         .activationPackage;
+
+      nixos =
```

Comment:
```suggestion
      nixos-module =
```

And similarly, `home-manager-module` above.
I feel like it's more explicit.

Reply:
Done. Also disabled the nixos checks for darwin